### ISSUE TITLE
Image Output Quality

### ISSUE THREAD

**Issue by Dalick** [Gulix/geckos#119]
Is it possible to control the resolution of the image when a card is saved as a .PNG? Currently, the .PNG images always seem to be created at a resolution of 96 pixels/inch regardless of the resolution of the image elements that were used when designing the card. I would like to export the cards as .PNG files with a resolution of 300 pixels/inch so that they look better when printed.

I saw that issue #29 is going to add PDF support, but it was not clear if you could use that to export higher resolution card images.

If this is not something that is currently possible, is this something you have considered for a future release?

**Comment by Gulix** [comment]
It's something I did not considered at all. I don't know if it's possible, but I will look into FabricJS (the library that generates the image) to see if it's possible.

**Comment by Dreadaxe** [comment]
Le format PNG peut être édité avec Adobe Fireworks. Est-ce-que c'est le cas Des PNG de Geeckos ?

**Comment by Gulix** [comment]
@Dreadaxe : Oui, le PNG est un format standard, de type Bitmap (non étirable). Il est modifiable par tout type d'éditeur d'image. Mais la qualité de l'image n'a rien à voir avec ce format. Il faudra sans doute pousser vers le format SVG.

> PNG Format can be edited by Adobe Fireworks. Are Geckos' PNG editable this way ?

> Yes, PNG is a standard format, of Bitmap type (not scalable). It can be modified by any type of image editor. But Image Quality is not related to this format. The SVG format should be the solution to look at.

**Comment by Dreadaxe** [comment]
Ce que je veux dire c'est que j'ai des fichiers PNG qui contiennent des
informations de font et des calques.

**Comment by Gulix** [comment]
http://superuser.com/questions/50044/how-to-read-fireworks-png-multilayer-files-without-fireworks

It appears it's something specific to Adobe Fireworks, in a proprietary format. So I will not go that way.
